dprint.json

```diff
@@ -17,16 +16,11 @@
     "singleQuote": true
   },
   "includes": ["**/*.{json,json5,ts,cjs,js,mjs,md,yml}"],
-  "excludes": [
-    ".git",
-    "lib",
-    "bin",
-    ".ameba.yml"
-  ],
+  "excludes": [".git", "lib", "bin", ".ameba.yml"],
```

Comment:
I don't know which commit made this change... Is this? https://github.com/dprint/dprint/pull/580 🤔